install

```diff
@@ -82,6 +82,7 @@ end
 def wait_for_user
   puts
   puts "Press RETURN to continue or any other key to abort"
+  puts "If the program is stopping here, try gcc to accept Xcode command-line tools"
```

Comment:
Shouldn’t that be somewhere else than in `wait_for_user`? This method should "wait for user" and nothing else IMHO.